Extensions/timestamps.js

```diff
@@ -149,11 +155,16 @@ XKit.extensions.timestamps = new Object({
 
 			var blog_name = '';
 			if (XKit.interface.where().inbox !== true) {
-				if (post.find('.permalink').length <= 0 && post.find(".post_permalink").length <= 0) {
+				var permalink = '';
+				if ((post.find('.permalink').length <= 0 && post.find(".post_permalink").length <= 0) && (post.find(".post-info-tumblelog").find("a").length <= 0)) {
```

Comment:
should be `post.find('.permalink, .post_permalink, .post-info-tumblelog a')` and we should store the result to use below